### PR TITLE
Fix language and sorting filters

### PR DESCRIPTION
**Pull request by BetaHuhn**
This PR fixes the language and sorting filters (#179)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,7 +87,7 @@ <h1 class="text-center text-5xl font-bold mb-8">Find issues to help with during
                                 >Newest</option>
                                 <option
                                     selected
-                                    value="newest"
+                                    value="noReplys"
```
Comment (damcclean) [inline index.html:90]: Could you change this to `noReplies` please?

File: src/js/app.js [hunk 2]
```diff
@@ -31,8 +30,8 @@ new Vue({
                 .then(() =>
                     fetch(
                         `https://api.github.com/search/issues?page=${this.page
-                        }&q=language:${this.filterLanguage
-                        }+label:hacktoberfest+type:issue+state:open+${this.noReplyOnly &&
+                        }&q=language:${this.selectedLanguage
+                        }+label:hacktoberfest+type:issue+state:open+${this.selectedSort === 'noReplys' &&
```
Comment (damcclean) [inline src/js/app.js:34]: Also needs changing here... `noReplies`

**Review by damcclean — CHANGES_REQUESTED** [review]
Looks good, thanks for the pull request!

Just a few little changes and I'll be good to merge.

File: index.html (hunk 1 above)
Comment (BetaHuhn) [reply index.html:90]: Ups, of course!

**Comment by damcclean** [comment]
Awesome, thanks for making the changes!

**Pull request merged**